packages/components-js/tests/unit/fixtures/stats.json

```diff
@@ -168,6 +169,7 @@
     "chunkShortName": "tabs-bar",
     "size": 8415,
     "gzipSize": 3326
+
```

Comment:
Please revert new lines.  
Also why didn't `text-field-wrapper` become smaller?

Reply:
I think this came from the failed merge yesterday, sorry 🥺 text-field-wrapper actually became smaller now